backend/app/controllers/products.js

```diff
@@ -1470,15 +1467,24 @@ function* checkProductData(req) {
             );
         }
     }
-
-    //TODO: Add check here to point to survey service
-    // if (req.body.surveyId) {
-    //     var isExistSurvey = yield thunkQuery(Survey.select().where(Survey.id.equals(req.body.surveyId)));
-    //     if (!_.first(isExistSurvey)) {
-    //         throw new HttpError(403, 'Survey with id = ' + req.body.surveyId + ' does not exist');
-    //     }
-    // }
-
+    //Check survey service is the survey exist in there
+    const path = 'surveys/';
+
+    const requestOptions = {
+        url: config.surveyService + path + req.body.surveyId,
+        method: 'GET',
+        headers: {
+            'authorization': req.headers.authorization
+        }
+    };
+    request(
```

Comment:
Since I do this in two places, I need to make it a function in common and call the function from here.